### ISSUE TITLE
Manual Installation in TextMate2

### ISSUE THREAD

**Issue by pilarArr** [MikeMcQuaid/GitHub-Markdown.tmbundle#22]
The manual [installation](https://github.com/MikeMcQuaid/GitHub-Markdown.tmbundle/blame/59938ad82921c6ccc3ee5f0ab688d2bc090edeaa/README.md#L16-L20) in TextMate 2 does not longer work. The bundles are not located in 
`/Application Support/Avian/`. Supposedly new bundles are located in `/Application Support/TextMate/Pristine Copy/Bundles`  but that doesn't work either

**Comment by noniq** [comment]
The current path for manually installed bundles is `~/Library/Application Support/TextMate/Bundles`. I’ll update the README accordingly, thanks for noticing!

**Comment by pilarArr** [comment]
Hi @noniq, I tried installing the bundle there but it doesn't work either, the path `~/Library/Application Support/TextMate/Bundles` is according to [TextMate specs](https://manual.macromates.com/en/bundles) the path for the diff between the installed bundles and any changes you made to them.

 Somewhere in the bottom it says that you should install new bundles in  `/Library/Application Support/TextMate/Bundles` and not in `~/Library/...` path but I haven't checked. I managed to get it working by installing it in `~/Library/Application Support/TextMate/Managed/Bundles`.

**Comment by noniq** [comment]
`~/Library/Application Support/TextMate/Bundles` is used for customizations (diffs) of pristine bundles as well as for complete bundles. `/Library/…` works the same but affects TextMate for all user accounts instead of just your own. Finally, `~/Library/Application Support/TextMate/Managed/` is managed by TextMate and used to store the bundles you install via checking them in TextMate’s preferences. You should not change / add bundles there.

TL;DR: Installing into `~/Library/Application Support/TextMate/Bundles` is the correct and recommended way, I’ve updated the readme accordingly. If it didn’t work for you there must have been another issue. Have you tried [reverting everything to defaults](https://github.com/textmate/textmate/wiki/Reverting-To-Defaults)?
